src/components/Talent/YourLinks.tsx

```diff
@@ -146,7 +146,7 @@ export function YourLinks({ success, useFormStore }: Props) {
   const { isOpen, onOpen, onClose } = useDisclosure();
   const { form } = useFormStore();
   const [pow, setPow] = useState<PoW[]>([]);
-  const [socialsError, setsocialsError] = useState<boolean>(false);
+  const [socialsError, setsocialsError] = useState<number | boolean>(false);
```

Comment:
keeping your idea in mind, i dont think you'd need a `boolean` type 
or initialize the state with `false`; initializing with 0 would be better in my opinion

Reply:
That was my initial thought too. Though 0 is a falsy value, `0===false` equates to false, which was why I was skeptical.
However I think it wouldn't matter in this context, let's try checking